backend/clubs/views.py

```diff
@@ -2691,6 +2692,20 @@ def get_serializer_class(self):
         return UserClubVisitSerializer
 
 
+class SearchViewSet(viewsets.ModelViewSet):
+    """
+    list: Return a list of search queries
+
+    create: Add a new search query
+    """
+
+    permission_classes = [IsAuthenticated]
+    http_method_names = ["post"]
+
+    def get_serializer_class(self):
+        return SearchSerializer
```

Comment:
We don't want to give a list of every search query to anyone who requests it. People should be able to create search queries, but only superusers should be able to list them.

Reply:
In this case, you document a `list` endpoint, but the user can't actually list stuff. Just delete this comment and you should be fine.